.github/workflows/triage.yml

```
@@ -86,8 +86,8 @@ jobs:
 
             - label: no ARM bottle
               path: Formula/.+
-              content: '\n    sha256.* big_sur: +"[a-fA-F0-9]+"\n'
-              missing_content: '\n    sha256.* arm64_big_sur: +"[a-fA-F0-9]+"\n'
+              content: '\n    sha256.* (?!.*(?:arm64_|_linux)).+: +"[a-fA-F0-9]+"\n'
+              missing_content: '\n    sha256.* arm64_(.+): +"[a-fA-F0-9]+"\n'
```

Didn't notice this initially but:

```suggestion
              missing_content: '\n    sha256.* arm64_.+: +"[a-fA-F0-9]+"\n'
```